### DISCUSSION TITLE
Feature request: Line item overrides

### DISCUSSION THREAD

**Discussion by BrandoCommando** [Shopify/function-examples#263] [Feedback: Cart Transform API]
Currently my merchant offers product upgrades that show as a separate product on the order, which get combined with the main product line in liquid:
<img width="707" alt="image" src="https://github.com/Shopify/function-examples/assets/97893/2bff52a9-0826-4631-81a3-89ffda2d48f3">

I am able to get _almost_ the same feel with the Cart Transform merge operation, but I'm still left with the default image box:
<img width="445" alt="image" src="https://github.com/Shopify/function-examples/assets/97893/778c71fe-6075-4a16-a5f3-48fb76344708">

It would be nice if there was a image/title (and subtitle) override for individual child line items. It would also be nice if images could be null to remove the placeholder altogether. Currently an InvalidOutputError occurs when setting image to null.

**Comment by ignacio-chiazzo** [comment]
Hey, Thanks for the feedback. 

I just wanted to say that we are considering this and the other suggestions you submitted. Please keep adding them!

**Comment by mumarse** [comment]
@BrandoCommando  have you got any solution for this
